torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -2476,6 +2478,96 @@ Error PyTorchModelLoader::loadListConstruct(const torch::jit::Node *ptNode) {
   RETURN_ERR(addValueMapping(outputs[0], std::move(glowIVal)));
 }
 
+Error PyTorchModelLoader::loadListUnpack(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+
+  std::vector<NodeValue> *inputNodeVals;
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      inputNodeVals, iValToNodeValueList(getGlowIValueForValue(inputs[0])));
+
+  RETURN_ERR_IF_NOT(inputNodeVals->size() == outputs.size(),
+                    "[ListUnpack] Input size must equal output size");
+
+  for (size_t i = 0; i < inputNodeVals->size(); ++i) {
+    RETURN_IF_ERR(addValueMapping(outputs[i], (*inputNodeVals)[i]));
+  }
+
+  return Error::success();
+}
+
+Error PyTorchModelLoader::loadSplit(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 3, outputs, -1));
+
+  glow::NodeValue input;
+  if (hasGlowNodeValueForValue(inputs[0])) {
```

Comment:
Please add explicit name for each input, like, inputs[SplitInput::input].
You can find examples in other ops.

Reply:
Added in newest commit